docs/Support-Tiers.md

```diff
@@ -63,6 +63,7 @@ Tier 2 configurations include:
 - using official packages that need to be built from source due to installing Homebrew outside the default prefix
   (i.e. `/opt/homebrew` on Apple Silicon, `/usr/local` on Apple Intel x86_64, `/home/linuxbrew/.linuxbrew` for Linux)
 - running on a not-yet-supported architecture (i.e. Linux ARM64/AARCH64)
+- devices using OpenCore Legacy Patcher
```

Comment:
```suggestion
- devices using OpenCore Legacy Patcher with a Westmere or later Intel CPU
```

Technically more complex with that (Haswell only matters if macOS >= 13), but macOS < 13 is tier 3 anyway so we can keep it simple here.

May become more complex again in the future if arm64 becomes relevant here.

Reply:
We only check for the existence of OCLP, so I kept with that assumption. Maybe we can actually update the check to catch this and change the docs accordingly?
https://github.com/Homebrew/brew/blob/213633371cdf5bd7713baf78ec8d0ce9a8593af5/Library/Homebrew/extend/os/mac/diagnostic.rb#L134-L154

Reply:
We don't really have CPU comparison functions as it's kinda tricky to do. Ideally we'd compare with `Hardware.oldest_cpu`.

Though if we don't care about macOS <13, I guess a check for PCLMUL might be a simplistic way to do so this.